resources/common.js

```diff
@@ -212,24 +212,46 @@ function count_down(label) {
     }, 1000);
 }
 
-function register_time(elems, limit) {
-    limit = limit || 300;
+function set_date_locale(language_code) {
+    try {
+        var rtf = new Intl.RelativeTimeFormat(language_code);
+        window.format_ms = function (amount) {
+            amount = Math.trunc(amount / 1000); // seconds
+            if (Math.abs(amount) < 120) return rtf.format(amount, 'second');
+            amount = Math.trunc(amount / 60);   // minutes
+            if (Math.abs(amount) < 180) return rtf.format(amount, 'minute');
+            amount = Math.trunc(amount / 60);   // hours
+            if (Math.abs(amount) < 48) return rtf.format(amount, 'hour');
+            amount = Math.trunc(amount / 24);   // days
+            if (Math.abs(amount) < 100) return rtf.format(amount, 'day');
+            return '';          // beyond 100 days, use absolute time
+        };
+        window.format_ms(0);    // check for errors
```

Comment:
Should the check for support be something like

```js
typeof Intl !== "undefined" && !!Intl.RelativeTimeFormat
```

so we don't rely on errors?

Reply:
Done. Chrome and Edge go to the `if` block. IE11 goes to the `else` block.